src/app/admin/backup.ts

```diff
@@ -63,7 +63,18 @@ export const uploadBackup =
             addEventToCurrentSpan("Static channel backup to Dropbox successful.")
           } catch (error) {
             logger.error({ error }, "Static channel backup to Dropbox failed.")
-            recordExceptionInCurrentSpan({ error: error, level: ErrorLevel.Warn })
+            if (error instanceof Error) {
```

Comment:
couldn't `recordExceptionInCurrentSpan` be changed to accept `Error | unknown` and pass `"Static channel backup to Dropbox failed."` as a third `fallbackMsg:` arg?